experimental/common-expression-language/README.md

```diff
@@ -42,28 +50,30 @@ See [here](./api-reference.md) for API reference.
 |\	        |Escape character for templates, expressions.                                               |N/A                                |
 
 ## Variables
-Variables are always referenced by their name. E.g. @{myVariable}
-Variables can be complex objects. In which case they are referenced either using the property selector operator e.g. myParent.myVariable or using the item index selection operator. E.g. myParent.myList[0]. or using the [parameters](TODO) function. 
 
-There are two special variables, `[]` and  `{}`.
-`[]` represents an empty list, `{}` represents a empty object.
+Variables are always referenced by their name in the format `@{myVariable}`. Variables can be referenced either using the property selector operator in the form of `myParent.myVariable`, using the item index selection operator like in `myParent.myList[0]`, or using the [parameters](TODO) function. 
```

Comment:
The format for referring to expressions is change to `${expression}` instead of `@{expression}`